plugins/Feedback/lang/en.json

```diff
@@ -38,7 +39,7 @@
         "TopLinkTooltip": "Tell us what you think, or request Professional Support.",
         "ViewAnswersToFAQ": "View answers to %1$sFrequently Asked Questions%2$s",
         "FAQs": "FAQs",
-        "ViewUserGuides": "Learn how to configure Matomo and how to effectively analyze your data with our %1$suser guides%2$s",
+        "ViewUserGuides": "FAQs, comprehensive guides, how-to’s and troubleshooting documents.",
```

Comment:
We can't replace a translation string with placeholders with one without. We need to create a new key for the new translation string without the placeholder, and remove the old one if no longer needed. This way weblate will handle the translations correctly for other languages.

Reply:
Thank you. I just reverted a couple translations and created new ones instead.